x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/rule_params_modifier.ts

```diff
@@ -11,6 +11,7 @@ import type { BulkActionEditForRuleParams } from '../../../../../common/detectio
 import { BulkActionEditType } from '../../../../../common/detection_engine/schemas/common/schemas';
 
 import { invariant } from '../../../../../common/utils/invariant';
+import { isMachineLearningParams } from '../../signals/utils';
```

Comment:
can we add a functional test to check this behaviour and prevent for happening it in future?
Here, we have a bunch of tests that tests some basic and corner cases:
https://github.com/elastic/kibana/blob/main/x-pack/test/detection_engine_api_integration/security_and_spaces/group1/perform_bulk_action.ts#L364

Thanks